### PR TITLE
Update create-a-phone-system-call-queue.md

### PR DESCRIPTION
**Pull request by dehulme**
This public documentation does not have any reference to Hybrid scenarios, this only describes the steps that should be followed for Microsoft Teams, even though at the beginning of the page it's mentioned: "Applies to: Skype for Business, Microsoft Teams"

Customer requested to be added a reference in this article where it is stated that the resource account in hybrid scenario needs to be created on-premises, and a reference to this article as well: https://docs.microsoft.com/en-us/skypeforbusiness/hybrid/plan-call-queue

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-call-queue.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ To set up a call queue, in the Teams admin center, expand **Voice**, click **Cal
 
 3. Choose a [supported language](create-a-phone-system-call-queue-languages.md). This language will be used for system-generated voice prompts and voicemail transcription (if you enable them).
 
+> [!NOTE]
+> In a Hybrid scenario, the resource account needs to be created on-premises. For more information, see [Plan Could call queues](https://docs.microsoft.com/en-us/skypeforbusiness/hybrid/plan-call-queue)
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:57]:
```suggestion
> In a hybrid scenario, the resource account needs to be created on-premises. For more information, see [Plan Cloud call queues](https://docs.microsoft.com/skypeforbusiness/hybrid/plan-call-queue).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@dehulme - please commit my suggestions.

**Comment by dehulme** [comment]
Good Morning

I believe the documentation has already been updated. Thanks a lot for coming back to me.

Delia

**Comment by MikePlumleyMSFT** [comment]
This info was added to [Plan for Teams auto attendants and call queues](https://docs.microsoft.com/microsoftteams/plan-auto-attendant-call-queue).

**Pull request closed without merging**